user_guide/configuration.rst

```diff
@@ -13,8 +13,16 @@ profiles.
 -------------
 
 All configuration happens inside a single configuration file in the ``YAML``
-format. Behat tries to load ``behat.yml`` or ``config/behat.yml`` by default,
-or you can tell Behat where your config file is with the ``--config`` option:
+format. By default, Behat loads the configuration from the first file matching:
+
+* ``behat.yaml`` or ``behat.yml``
+* ``behat.yaml.dist`` or ``behat.yml.dist``
+* ``behat.dist.yaml`` or ``behat.dist.yml``
+* ``config/behat.yaml`` or ``config/behat.yml``
+* ``config/behat.yaml.dist`` or ``config/behat.yml.dist``
+* ``config/behat.dist.yaml`` or ``config/behat.dist.yml``
```

Comment:
should this be an ordered list instead of an unordered list ?

Reply:
I mean.... now that you're saying it, it obviously makes sense. 😆

Reply:
Done: https://github.com/Behat/docs/pull/166